graphql/src/schema/meta.graphql

```diff
@@ -55,6 +55,8 @@ type _Block_ {
   hash: Bytes
   "The block number"
   number: Int!
+  "Timestamp of the block if available, format depends on the chain" 
```

Comment:
This query is deterministic, and we don't have good control of whether the timestamp is available.

An idea to better control this is: Make this field non-nullable, and return a new `QueryExecutionError` variant if it is requested but we can't produce it, and make sure that error returns `false` from `is_attestable`.

Reply:
This query is already non deterministic because it will return different results for the same block depending on whether the request was done using hash or number. This behaviour we're introducing is consistent with the behaviour of hash. I'm not sure introducing an error for a nullable field is going to produce the UX since it's not trivial to understand the different without being familiar with the code at some level.